core/src/components/modal/modal.tsx

```diff
@@ -623,6 +650,43 @@ export class Modal implements ComponentInterface, OverlayInterface {
     return eventMethod(this.el, 'ionModalWillDismiss');
   }
 
+  /**
+   * Move a sheet style modal to a specific breakpoint.
```

Comment:
Can we add a note that the breakpoint must be defined in your `breakpoints` array?